site/src/pages/RoadmapPage/Header.tsx

```diff
@@ -18,10 +19,23 @@ interface HeaderProps {
 }
 
 const Header: FC<HeaderProps> = ({ courseCount, unitCount, saveRoadmap }) => {
-  const show = useAppSelector((state) => state.transferCredits.showTransfersMenu);
+  const showTransfers = useAppSelector((state) => state.transferCredits.showTransfersMenu);
   const dispatch = useAppDispatch();
 
-  const toggleTransfers = () => dispatch(setShowTransfersMenu(!show));
+  const toggleTransfers = () => {
+    if (showTransfers) {
+      // After closing the menu, clear all the unread markers
+      dispatch(clearUnreadTransfers());
+    }
+    dispatch(setShowTransfersMenu(!showTransfers));
+  };
+
+  const unreadTransfers = useAppSelector((state) => state.transferCredits.unreadTransfers);
+
+  const showUnreadTransfers =
```

Comment:
Update variable naming so that it is clear that we mean "they exist" rather than "it should be displayed"
```suggestion
  const hasUnreadTransfers =
```